### ISSUE TITLE
Discussion and reasoning on license issue

### ISSUE THREAD

**Issue by sgauravm** [UBC-MDS/edapython#93]
- Discuss and reason the license choice by opening issues in both Python and R repositories.
- Include a link of these issues in your final_project repository's README.md file.

**Comment by marvinmin** [comment]
I think the MIT license we are using now is suitable for our project. Because we would like our package to help as many data scientists as it can and allow them to use this package in their projects without many constraints. And we also welcome people to modify the package to make it more helpful. The MIT license is a simple permissive license that allows users to use, copy, modify,  and distribute our package.

**Comment by sweber15** [comment]
I agree with you @marvinmin. Using [choosealicense.com](https://choosealicense.com/), an MIT license is "simple and permissive" which we would want for this project. We want people to be collaborative and allow our users to use it in their projects (do what they want with it) whether it be for commercial or private use.

**Comment by v5y8** [comment]
minimal viable license. I like it.

**Comment by sgauravm** [comment]
I agree with all as we developed this package for educational purpose and wanted limited restriction. The MIT license is a short and simple permissive license only requiring preserving of copyright and license notices. This correctly fits into our requirement so we all agreed to use the [MIT](https://choosealicense.com/licenses/mit/) license
